src/addons/messagelog/messagelog-archiver/src/main/java/ee/ria/xroad/messagelog/archiver/LogArchiver.java

```diff
@@ -100,7 +99,7 @@ private void markArchived(Session session, List<Long> recordIds) {
 
     private boolean handleArchive(long maxRecordId) throws Exception {
         return doInTransaction(session -> {
-            final int limit = getArchiveTransactionBatchSize();
+             final int limit = getArchiveTransactionBatchSize();
```

Comment:
random space in front? :)

Reply:
yes, to check if reviewer is actually reviewing :D